tests/codemods/test_subprocess_shell_false.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from core_codemods.subprocess_shell_false import SubprocessShellFalse
+from tests.codemods.base_codemod_test import BaseCodemodTest
+
+each_func = pytest.mark.parametrize(
+    "func", ["check_output", "check_call", "run", "call", "Popen"]
+)
+
+
+class TestSubprocessShellFalse(BaseCodemodTest):
+    codemod = SubprocessShellFalse
+
+    def test_name(self):
+        assert self.codemod.name() == "subprocess-shell-false"
+
+    @each_func
+    def test_import(self, tmpdir, func):
+        input_code = f"""import subprocess
```

Comment:
Test cases now automatically adjust indentation so this is much more readable if rendered like this:
```python
    input_code = f"""
    import subprocess
    subprocess.{func}(args, shell=True)
    """
```